1.8/tutorial/loadbalancing.md

```diff
@@ -0,0 +1,41 @@
+
+---
+post_title: DC/OS 101 - Loadbalancing
+menu_order: 3
+---
+
+# Prerequisites
+We expect you to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, you have just deployed app2 and marathon-lb in our cluster and have verified both are running.
+
+# Objective
+In the last session, we have learned how to scale our application.
+But if there multiple instances of an app in my cluster, how do cluster internal services or external request decide which instance should be used?
+
+# Steps
+  This is the topic of load balancing. With DC/OS you have two different inbuilt options: [Marathon-LB](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/) and [Named virtual IPs](https://dcos.io/docs/1.8/usage/service-discovery/load-balancing-vips/).
```

Comment:
+  Load-balancers decide which instance of an app internal or external services should use. With DC/OS you have two different built-in load-balancer options: [Marathon-LB](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/) and [Named virtual IPs](https://dcos.io/docs/1.8/usage/service-discovery/load-balancing-vips/).